deploy/local/vars_fbsd.sh

```diff
@@ -88,8 +88,8 @@ if [ ! -d $__daal_tmp_dir ]; then
     __daal_tmp_dir=$(dirname -- "$(get_script_path "$vars_script_name")")
 fi
 
-export DAL_MAJOR_BINARY=1
-export DAL_MINOR_BINARY=0
+export DAL_MAJOR_BINARY=__DAL_MAJOR_BINARY__
+export DAL_MINOR_BINARY=__DAL_MINOR_BINARY__
```

Comment:
for master it should be also bumped to 1.1

Reply:
Fixed, minor version was increased